.github/workflows/stale.yml

```
@@ -0,0 +1,35 @@
+name: Close stale issues and PRs
+
+permissions:
+  issues: write
+
+on:
+  schedule:
+    - cron: "20 2 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v6
+        with:
+          # For issues: post a "warning" message after 30 days, then close if another 30 days pass without a response. In another workflow, issues closed for 30 days will be locked.
+          stale-issue-message: "Hi there! 👋 We haven't heard from you in 30 days and would like to know if the problem has been resolved or if you still need help. If we don't hear from you before then, I'll auto-close this issue in 30 days."
+          close-issue-message: "I'm closing this issue because we haven't heard back in 60 days. ⌛️ If you still need help, feel free to comment or reopen the issue!"
+          days-before-issue-stale: 30
+          days-before-issue-close: 60
```

Sorry, I misunderstood the API, it seems like adding the stale label is an activity so 30+30=60

```suggestion
          days-before-issue-close: 30
```